admin/views/school/manage-pupils.ejs

```diff
@@ -46,7 +46,7 @@
         <div class="column-one-half">
             <h2 class="heading-medium heading-border-top" tabindex="0">Pupils to take check</h2>
 
-            <form method="post" action="/school/generate-pins" autocomplete="off">
+            <form method="post" action="/school/\-pins" autocomplete="off">
```

Comment:
Is this form action correct?

Reply:
that was a typo but this view along with the route will be eventually removed as it is not being used.